app/forms/answer_form.rb

```diff
@@ -1,6 +1,6 @@
 class AnswerForm
```

Comment:
What profit do you get when using it?

Reply:
When I use answer_form, attachment and answer add to database after create  answer